pom.xml

```diff
@@ -12,7 +12,7 @@
 	<name>YamlBeans</name>
 	<groupId>com.esotericsoftware.yamlbeans</groupId>
 	<artifactId>yamlbeans</artifactId>
-	<version>1.12</version>
+	<version>0.0.1-SNAPSHOT</version>
```

Comment:
Shouldn't this be `1.13-SNAPSHOT`?

Reply:
common practice is to use 0.0.1-SNAPSHOT during dev.
this is required for NEXUS support.
the version is then bumped at release time.

Reply:
I've never heard of that, and it seems more useful to indicate *which* version development will become when released, but I don't use Maven so it doesn't matter to me either way. :)

Reply:
I agree with you, but it so happens that a lot of maven related tools follow a specific behaviour based on this particular 0.0.1-SNAPSHOT version.
Quite annoying, but what can we do...

Reply:
I checked with Nexus, and it seems you are right i.e. it suffices to have -SNAPSHOT at the end to deploy to staging